core/meta/src/TClass.cxx

```diff
@@ -1513,8 +1513,9 @@ void TClass::Init(const char *name, Version_t cversion,
          if (proto)
             proto->FillTClass(this);
       }
-      if (!fHasRootPcmInfo && gInterpreter->CheckClassInfo(fName, /* autoload = */ kTRUE)) {
-         gInterpreter->SetClassInfo(this);   // sets fClassInfo pointer
+      TDictionary::DeclId_t decl {};
+      if (!fHasRootPcmInfo && gInterpreter->CheckClassInfo(fName, decl, /* autoload = */ kTRUE, /* instantiateTemplate */ kTRUE)) {
```

Comment:
The parameter name in the call do not match the parameter name in the function declaration (i.e. this call is not doing what the comment says it should be doing).

Reply:
sorry, good catch.